### PR TITLE
Add support to import Blocklist

### PR DESCRIPTION
**Pull request by humrochagf**
First iteration of blocklist support

- Adds support to simple domain list file (one domain per line)
- Adds support to Oliphant blocklist format (https://writer.oliphant.social/oliphant/the-oliphant-social-blocklist)

In this version it blocks all domains in the list since we don't have yet support to globally silencing domains.

relates to #613

### CODE REVIEW

File: users/views/admin/federation.py [hunk 1]
```diff
@@ -78,3 +81,58 @@ def get_initial(self):
...
+                try:
+                    validator(domain)
+                except ValidationError:
+                    # skip adding invalid domain
+                    # to the blocklist
+                    continue
+
+                domains.append(domain)
+        except (TypeError, ValueError):
+            return redirect(".?bad_format=true")
```
Comment (andrewgodwin) [inline users/views/admin/federation.py:124]: You can now use the Django messages framework for this instead - `from django.contrib import messages; messages.error(request, "Your CSV was...")`. I switched some of the other error messages so far but not all.

**Review by andrewgodwin — COMMENTED** [review]
Looks good overall, just one change to the way messages are shown if you could!

File: users/views/admin/federation.py (hunk 1 above)
Comment (humrochagf) [reply users/views/admin/federation.py]: Done :slightly_smiling_face:

**Review by andrewgodwin — APPROVED** [review]

**Comment by andrewgodwin** [comment]
Thanks for you work on this!

**Pull request merged**